libs/domain/search/facet-value-navigation/facet-value-navigation.component.ts

```diff
@@ -26,11 +26,12 @@ export class SearchFacetValueNavigationComponent
   @property({ type: Boolean }) enableSearch?: boolean;
   @property({ type: Boolean }) open?: boolean;
   @property({ type: Boolean }) enableClear?: boolean;
+  @property({ type: Boolean }) dirty?: boolean;
```

Comment:
mb better name `touched`?

Reply:
`touched` usually means some interaction with element, first focus or click, even without changing the values. dirty is more about differences between current and initial values

Reply:
and how it's possible to be dirty without interaction?

Reply:
I meant that `touched` state means that user 'touched' the input, focused, clicked on it and etc, this 
does not guarantee that values were changed. `dirty` means that values were changed according the initial ones. In this case I need to make some emphasis that values were changed - thats why I believe that dirty is more clear and suitable

Reply:
Or maybe do you have better ideas instead of `touched`?